[DependencyInjection] Update debug.rst

aliz71 opened the pull request:
The service ID should not be surrounded by a single quote.

<!--

If your pull request fixes a BUG, use the oldest maintained branch that contains
the bug (see https://symfony.com/releases for the list of maintained branches).

If your pull request documents a NEW FEATURE, use the same Symfony branch where
the feature was introduced (and `5.x` for features of unreleased versions).

-->

aliz71 committed:
Update debug.rst

The service ID should not be surrounded by a single quote.

javiereguiluz commented:
Did you experience some issue because of these quotes?

If I remember correctly, we added these quotes (here and in other commands across the docs) because not adding them causes problems for Windows users.

aliz71 commented:
Yes, when I added these quotes I got an error message in console and when I
executed without quotes the error was gone. By the way, my develop
environment is win 10.

On Fri, Sep 17, 2021, 8:15 PM Javier Eguiluz ***@***.***>
wrote:

> Did you experience some issue because of these quotes?
>
> If I remember correctly, we added these quotes (here and in other commands
> across the docs) because not adding them causes problems for Windows users.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/symfony/symfony-docs/pull/15820#issuecomment-921898516>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTOAYDH4N2ZSJ2NB2JH7BLUCNPAXANCNFSM5EBX5JJA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

javiereguiluz commented:
Thanks for the feedback! This is now more confusing to me. I remember that when we add unneeded quotes in some command it's because someone using Windows asked us to do that because otherwise it fails.

aliz71 commented:
You are welcome. By the way, I use without quotes it seems when I add the
quotes the framework thinks it is a part of my service name.
I attached an example for you to this email.

On Wed, 22 Sept 2021 at 16:40, Javier Eguiluz ***@***.***>
wrote:

> Thanks for the feedback! This is now more confusing to me. I remember that
> when we add unneeded quotes in some command it's because someone using
> Windows asked us to do that because otherwise it fails.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/symfony/symfony-docs/pull/15820#issuecomment-924915637>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTOAYHZ2FFYM3OMDP6ILXTUDHIVVANCNFSM5EBX5JJA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

javiereguiluz commented:
I've asked in https://symfony.com/slack and many developers using Windows replied. The summary is: if you use the basic `cmd` console, only the command WITHOUT quotes works; if you use any other advanced console/terminal, it works WITH and WITHOUT quotes.

So, we're merging this pull request because it makes it work for everybody. Thanks!

javiereguiluz commented:
Thank you Ali ... and congrats on your first Symfony Docs contribution :tada:

aliz71 commented:
Thanks for your respond. Good luck

On Wed, Sep 22, 2021, 5:19 PM Javier Eguiluz ***@***.***>
wrote:

> I've asked in https://symfony.com/slack and many developers using Windows
> replied. The summary is: if you use the basic cmd console, only the
> command WITHOUT quotes works; if you use any other advanced
> console/terminal, it works WITH and WITHOUT quotes.
>
> So, we're merging this pull request because it makes it work for
> everybody. Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/symfony/symfony-docs/pull/15820#issuecomment-924949275>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTOAYEEXKFET5QDXFZDNDLUDHNFHANCNFSM5EBX5JJA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
